src/features/singleRight/delegate/ChooseServicePage/ChooseServicePage.tsx

```diff
@@ -146,19 +148,19 @@ export const ChooseServicePage = () => {
             )}
             {filterChips()}
           </div>
-          <div className={classes.serviceResouces}> {serviceResouces}</div>
+          <div className={classes.serviceResources}> {serviceResources}</div>
           {totalNumberOfResults !== undefined && totalNumberOfResults > 0 && (
             <Pagination
               className={classes.pagination}
               currentPage={currentPage}
               totalPages={Math.ceil(totalNumberOfResults / searchResultsPerPage)}
               nextLabel={t('common.next')}
               previousLabel={t('common.previous')}
-              itemLabel={(num: number) => `Side ${num}`}
+              itemLabel={(num: number) => t('common.page') + ` ${num}`}
               onChange={setCurrentPage}
-              size='small'
+              size={isSm ? 'small' : 'medium'}
               compact={isSm}
-              hideLabels={isSm}
+              hideLabels
```

Comment:
Why not display the labels when on desktop? We have the room for it

Reply:
Thought people would confuse it with "Gå videre"-button

Reply:
Good point.